payment-handler/app-can-make-payment-true.js

```diff
@@ -0,0 +1,6 @@
+// Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang).
+
+// This app always responds to CanMakePaymentEvent with "true".
+self.addEventListener('canmakepayment', (canMakePaymentEvent) => {
+    canMakePaymentEvent.respondWith(true);
```

Comment:
All the things above would of course apply here too :)

Reply:
Roger, roger.